src/editor/manager.ts

```diff
@@ -151,15 +151,22 @@ export default class EditorManager {
     selection: Array<Selection['codeRef']['range']>
   ): Array<[number, number]> {
     if (!this._editorView) {
-      return selection.map((s): [number, number] => {
-        return [s[0], s[1]]
-      })
+      return selection
+        .filter((s) => s[2] === 0)
```

Comment:
We have a function for this:

https://github.com/KittyCAD/modeling-app/blob/d270447777582f97cbb27424ff7c30a4af5c556d/src/lang/util.ts#L30

Reply:
It would be nice to use the function because we've changed it a few times already, and we hope to change it again someday. It's annoying to search for `=== 0` but easy to change this function.

Reply:
Thank you!